sensor_msgs/Image keyframe_image
sensor_msgs/Image keyframe_depth
sensor_msgs/Image current_image
float32[4] intrinsics
float32[3] rotation_prior
float32[3] translation_prior
---
geometry_msgs/Transform transform
